contents/docs/error-tracking/_snippets/nuxt-install-error-tracking.mdx

```diff
@@ -14,14 +14,43 @@ Send errors directly using the posthog client, follow instructions on how to get
 
 ```vue component.vue
 <script>
-  function captureException() {
-    posthog.captureException(new Error("Error message"), {
-      ...additionalProperties
-    })
+  const { $posthog } = useNuxtApp()
+
+  if ($posthog) {
+    const posthog = $posthog()
+    posthog.captureException(new Error("Important error message"))
   }
 </script>
 ```
 
+On the server side, you can use the `posthog` object directly.
+
+```js file=server/api/example.js focusOnLines=4-21
+export default defineEventHandler(async (event) => {
+  const distinctId = getCookie(event, 'distinct_id')
+
+  const { PostHog } = await import('posthog-node');
+  const runtimeConfig = useRuntimeConfig()
+
+  const posthog = new PostHog(
+    runtimeConfig.public.posthogPublicKey,
+    { 
+      host: runtimeConfig.public.posthogHost, 
+      defaults: runtimeConfig.public.posthogDefaults,
+      capture_pageview: false
```

Comment:
Node doesn't capture pageviews and doesn't use the defaults. Those are both js web config options
```suggestion
```